src/Microsoft.OpenApi.Readers/V2/OpenApiHeaderDeserializer.cs

```diff
@@ -82,5 +150,23 @@ public static OpenApiHeader LoadHeader(ParseNode node)
 
             return header;
         }
+
+        private static void LoadStyle(OpenApiHeader h, string v)
+        {
+            switch (v)
+            {
+                case "csv":
+                    h.Style = ParameterStyle.Simple;
+                    return;
+                case "ssv":
+                    h.Style = ParameterStyle.SpaceDelimited;
+                    return;
+                case "pipes":
+                    h.Style = ParameterStyle.PipeDelimited;
+                    return;
+                case "tsv":
+                    throw new NotSupportedException();
+            }
```

Comment:
>   } [](start = 9, length = 4)

Looks like we should throw an error for case default? That would indicate that the document is broken in V2.